osdk/src/base_crate/x86_64.ld.template

```diff
@@ -117,11 +117,14 @@ SECTIONS
     # processor, while it would be copied to other dynamically allocated memory
     # areas for the application processors.
     . = ALIGN(4096);
+    __cpu_local_start = .;
+    .cpu_local_tss          : AT(ADDR(.cpu_local_tss) - KERNEL_VMA) {
```

Comment:
See the comments on the trapframe implementation for the ideas on the task state segment. There's no need for an extra `.cpu_local_tss` segment.

Also if you insist to do so, make the comments better. The above comments just a apply to the `.cpu_local` section. The readers will be confused for the `tss` sectrion.

Reply:
This segment is intend to make sure that `GSBASE` is pointed to `TSS` struct, so that we don't need to calculate its offset when do `syscall_entry`. This offset could be determined at compile time, but I can't think of a good way to get it.

Reply:
I see. You can leave the implementation as is. But remember to improve the comments.